usaspending_api/common/validator/award_filter.py

```diff
@@ -26,6 +26,8 @@
     {"name": "recipient_search_text", "type": "array", "array_type": "text", "text_type": "search"},
     {"name": "recipient_type_names", "type": "array", "array_type": "text", "text_type": "search"},
     {"name": "set_aside_type_codes", "type": "array", "array_type": "text", "text_type": "search"},
+    {"name": "object_class", "type": "array", "array_type": "text", "text_type": "search"},
+    {"name": "program_activity", "type": "array", "array_type": "integer", "array_max": maxsize},
```

Comment:
Here you specify that `program_activity` is an array of integers but then in your test data you provide an array of strings. Depending on which one is correct one or the other should be updated. 

Here is the test data taken from below:
```
def legacy_filters():
    return {"award_type_codes": ["A", "B"], "object_class": ["111"], "program_activity": ["222"]}
```

Reply:
good catch. A quirk of TinyShield is that it will work with either by trying to cast (think `==` instead of `===`), but I'd rather encourage strong typing in my examples

Reply:
> A quirk of TinyShield

It was an intentional design decision. Give API users the benefit of the doubt, if the string is only numeric typecast and continue.